tests/test_expect.py

```diff
@@ -43,3 +44,22 @@ def test_chaining(self):
         expect(expect)\
             .to.have.chain('to')\
             .to.have.chain('be')
+
+    def test_multiple_not_to_success(self):
+        expect(1).eq(1)
+        expect(1).not_to.not_to.eq(1)
+        expect(1).not_to.not_to.not_to.not_to.eq(1)
+
+        expect(1).to.ne(2)
+        expect(1).not_to.not_to.ne(2)
+        expect(1).not_to.not_to.not_to.not_to.ne(2)
+
+    @must_fail
+    def test_multiple_not_to_failure(self):
+        expect(1).not_to.eq(1)
+        expect(1).not_to.not_to.not_to.eq(1)
+        expect(1).not_to.not_to.not_to.not_to.not_to.eq(1)
+
+        expect(1).not_to.ne(2)
+        expect(1).not_to.not_to.not_to.ne(2)
+        expect(1).not_to.not_to.not_to.not_to.not_to.ne(2)
```

Comment:
Have to admit that reading this cause my little brain blows 😂, but it's fun figuring these chains.